project/Build.scala

```diff
@@ -57,6 +58,10 @@ object ApplicationBuild extends Build {
       "com.arpnetworking.metrics.extras" % "apache-http-sink-extra" % "0.6.0",
       "com.arpnetworking.metrics.extras" % "jvm-extra" % "0.4.2",
       "com.arpnetworking.metrics" % "metrics-client" % "0.7.0",
+      "com.chrisomeara" %% "pillar" % "2.3.0",
```

Comment:
I'm fine with this for now, but we need a better strategy for these various back-end implementation dependencies. Off the top of my head I'm thinking that we mark all such dependencies (e.g. Postgres, ES, Cass) as provided and disable their configs by default (I think done already) and then extenders can configure and provide the subset of db technologies they need. Thoughts?